src/schematools/importer/base.py

```diff
@@ -463,6 +470,21 @@ def is_versioned_dataset(self) -> bool:
             )
         return is_versioned
 
+    def create_schema(self, db_schema_name: str) -> None:
+        """Create DB Schema.
+
+        Is a no-op if schema already exits.
```

Comment:
exits -> exists

Reply:
Yeah, I do that often. Unfortunately I lost my access to this repo on Friday December 31st. Hence I can't make any changes anymore.